ui/easydiffusion/server.py

```diff
@@ -181,6 +186,25 @@ def render_internal(req: dict):
         log.error(traceback.format_exc())
         raise HTTPException(status_code=500, detail=str(e))
 
+def model_merge_internal(req: dict):
+    try:
+        from sdkit.train.merge_models import merge_models
```

Comment:
It should be enough to do `from sdkit.train import merge_models`. It'll import the function, instead of the submodule. This is the recommended way to import (declared in the API), since the inner modules are hidden (implementation details) and can change.